src/services/config.js

```diff
@@ -76,6 +77,10 @@ class ConfigData {
         // Clean the profile ID.
         profileId = this.sanitize(profileId);
         profile = this.getProfileFromConfigFileById(profileId);
+        // Explicitly add `id` to the returned profile
+        if (profile && !profile.hasOwnProperty('id')) {
```

Comment:
Better to move this code snippet after Line:86,  so that we don't need [these](https://github.com/twilio/twilio-cli-core/pull/124/files#diff-69b5d67d841057efec105a48299840e180ffc94f0da052f105dd16f5465d6a9cR115) changes.